contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -45,6 +49,34 @@ def get_available_memory_metric():
     return gauge
 
 
+def get_process_private_bytes():
+    try:
+        process = psutil.Process()
+        return process.memory_info().rss
+    except psutil.NoSuchProcess as ex:
+        logger.error('Error: Process does not exist %s.', ex)
+    except psutil.AccessDenied as ex:
```

Comment:
What about other [exceptions](https://psutil.readthedocs.io/en/latest/#exceptions) that might be added in the future?

Reply:
Good point. I'll add a generic catch.